website/server/controllers/api-v3/user.js

```diff
@@ -1267,32 +1267,6 @@ api.userReroll = {
 };
 
 /**
-* @api {post} /api/v3/user/addPushDevice Add a push device to a user
```

Comment:
Hmmm.... we're sure this could not actually be used, right? Otherwise, this is a breaking change.